www-apps/prowlarr/prowlarr-0.3.0.1730.ebuild

```
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+SRC_URI="
+	amd64? ( https://github.com/Prowlarr/Prowlarr/releases/download/v${PV}/Prowlarr.develop.${PV}.linux-core-x64.tar.gz )
+	arm? ( https://github.com/Prowlarr/Prowlarr/releases/download/v${PV}/Prowlarr.develop.${PV}.linux-core-arm.tar.gz )
+	arm64? ( https://github.com/Prowlarr/Prowlarr/releases/download/v${PV}/Prowlarr.develop.${PV}.linux-core-arm64.tar.gz )
+"
+
+DESCRIPTION="An indexer manager/proxy to integrate with your various PVR apps."
+HOMEPAGE="https://wiki.servarr.com/prowlarr"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64"
```

We know it can't work elsewhere unless the ebuild itself gets modified:
```suggestion
KEYWORDS="-* ~amd64 ~arm ~arm64"
```